recipes/dart/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright (c) 2012-2018 Red Hat, Inc.
```

Comment:
@sr229 Let me make sure. Are you working in Red Hat?

Reply:
Oh this was a copy of the header from the rest of the images (I based it off the Android stack). I can remove it if such.

Reply:
@sr229 I read `recipes/android/Dockerfile` and I think your code is almost "from scratch".
@eivantsov How do you think?